### PR TITLE
Feat(ARC-1307): Title text button background image block

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/ARC-1307

part of a group of PR's:
* https://github.com/viaacode/hetarchief-client/pull/844
* https://github.com/viaacode/react-admin-core-module/pull/169
* https://github.com/viaacode/avo2-components/pull/525

tekst left:
image
image

tekst right:
image
image

### CODE REVIEW

File: src/styles/admin-core/content-blocks/_image-text-background.scss [hunk 1]
```diff
@@ -36,17 +37,19 @@ $component: "c-block-image-text-background";
...
-			&--right {
-				.c-button__content {
-					flex-direction: row-reverse !important;
-				}
-			}
-		}
 	}
 
 	&__image-wrapper {
+		border: 1px solid red;
```
Comment (silkederudder-shd) [inline src/styles/admin-core/content-blocks/_image-text-background.scss:43]: rem

**Review by silkederudder-shd — APPROVED** [review]

File: src/styles/admin-core/content-blocks/_image-text-background.scss (hunk 1 above)
Comment (WardVerc1) [reply src/styles/admin-core/content-blocks/_image-text-background.scss:43]: Was this border requested? Probably used for testing no?

File: src/styles/admin-core/content-blocks/_image-text-background.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	&__heading {
+		margin-bottom: $spacer-sm !important;
+	}
+
+	&__button {
+		margin-top: $spacer-xs;
+	}
+
+	&__image-wrapper {
+		border: 1px solid red;
```
Comment (WardVerc1) [inline src/styles/admin-core/content-blocks/_image-text-background.scss:43]: Was this border requested? Probably used for testing no?

**Review by WardVerc1 — CHANGES_REQUESTED** [review]

File: src/styles/admin-core/content-blocks/_image-text-background.scss (hunk 2 above)
Comment (bertyhell) [reply src/styles/admin-core/content-blocks/_image-text-background.scss:43]: yes, i'll remove it

**Pull request merged**